blueocean-dashboard/src/main/js/components/RunDetailsPipeline.jsx

```diff
@@ -89,401 +33,93 @@ export class RunDetailsPipeline extends Component {
             document.addEventListener('keydown', this._handleKeys, false);
```

Comment:
... this way you don't have to worry about key presses or scrolling with the wheel or scrollbar, etc.